packages/cashier/src/pages/account-transfer/account-transfer.tsx

```diff
@@ -32,8 +56,8 @@ const AccountTransfer = ({
     setActiveTab,
     setIsTransferConfirm,
     setSideNotes,
-}) => {
-    const [is_loading_status, setIsLoadingStatus] = React.useState(true);
+}: TAccountTransferProps) => {
+    const [is_loading_status, setIsLoadingStatus] = React.useState<boolean>(true);
```

Comment:
@hamid-deriv mmm think that we can remove <boolean> generic because for primary types, we don't need to make the type explicit, since typescript can infer them. but it's just a suggestion